src/components/Problems/Discussion.tsx

```diff
@@ -123,7 +123,7 @@ export const Discussion: FC<DiscussionProps> = ({problemId, invalidateSeriesQuer
               return (
                 <Stack sx={comment.state !== CommentState.Published ? {color: 'gray'} : {}} key={comment.id}>
                   <Stack gap={1} direction="row" justifyContent={'space-between'}>
-                    <Typography variant="h3" component="span">
+                    <Typography variant="h3" component="span" style={{wordBreak: 'break-word'}}>
```

Comment:
sx I guess? ci nefunguje?
```suggestion
                    <Typography variant="h3" component="span" sx={{wordBreak: 'break-word'}}>
```